buildtools/plugin/src/main/kotlin/buildtools/AndroidAppPlugin.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * This Kotlin source file was generated by the Gradle 'init' task.
```

Comment:
This comment is not needed

Reply:
d3b17f0b4ae40a3d80f78a76aa38776c2696595a